development/tests/test_views_forms.py

```diff
@@ -50,3 +50,23 @@ def test_get_queryset_my_bots(self):
         request.user = self.user1
         response = MyBotsView.as_view()(request)
         self.assertEqual(response.status_code, 200)
+
+    def test_get_form_add_bot_get(self):
+        request = self.factory.get('development:addbot')
+        request.user = self.user1
+        response = AddBotView.as_view()(request)
+        self.assertEqual(response.status_code, 200)
```

Comment:
Have you seen this assert to add to the test?

```
self.assertTemplateUsed(response, 'development/add_bot.html')
```

Reply:
assertTemplateUsed are only usable on responses fetched using the Django test Client. We arent using that